prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskMaintenanceService.kt

```diff
@@ -27,7 +27,7 @@ class TaskMaintenanceService(
         logger.info("Rekjører ${tasks.size} tasks")
 
         tasks.forEach {
-            taskService.klarTilPlukk(it, TaskLogg.BRUKERNAVN_NÅR_SIKKERHETSKONTEKST_IKKE_FINNES)
+            taskService.klarTilPlukk(it.medTriggerTid(LocalDateTime.now()), TaskLogg.BRUKERNAVN_NÅR_SIKKERHETSKONTEKST_IKKE_FINNES)
```

Comment:
Mulig jeg er litt treig, men er ikke triggertid "tilbake i tid" her? Oppdaterer vi/dere triggertid til neste dag/kveld hvis en task feiler? 🤔 

Hvis man ser på feilede tasks så har de triggertid bakover i tid, praktisk for å se når noe begynte å feile. 

Kan vi isåfall kun oppdatere triggertid til "now" dersom forventet triggertid er fremover i tid? Og beholde hvis det tidspunktet har passert?

Reply:
Eks: https://familie-prosessering.intern.nav.no/service/familie-tilbake

Reply:
Men da har den jo ikke feilet ennå? 

Det er vel ikke feilet før "max antall feil" har blitt effektuert, og har du 1 døgn triggertid og maxAntallFeil = 3 - så sier du i praksis at du vil vente 3 døgn før du setter den til "feilet"..?

Reply:
Joda. Triggertid oppdateres når tasken blir satt til FEILET. Så tasken vil se ut sånn hvis den feiler 01.01.24 21:00
KLAR TIL PLUKK -  triggerTid=01.01.24 21:00
FEILET -  triggerTid=02.01.24 21:00
KLAR TIL PLUKK -  triggerTid=02.01.24 21:00   //settes til KLAR_TIL_PLUKK klokka 7

Reply:
Da skjønner jeg. Men vil du da egentlig ha triggertidPåFeilet i disse situasjonene? For fjerner du den så vil den jo rekjøre kl 7 neste dag og hver dag frem i tid...

Reply:
De har jo feilet. Vil bli rekjørt hver dag frem i tid. Eneste forskjellen er at man nå har mulighet til å se at de feiler uten å måtte jobbe nattt

Reply:
Mulig vi må ta en prat, for trodde den triggerTidFremITid skulle brukes for å "prøve på nytt senere" - og etter maks antall feil så prøver den ikke på nytt igjen og får ikke satt ny triggertid, men mulig det er noe der jeg misforstår...